kinds/preference/preference_kind.cue

```diff
@@ -0,0 +1,51 @@
+package kind
+
+name:     "Preference"
```

Comment:
I was a bit indecisive between `Preference` and `Preferences` but I decided over the former because [the model we currently use](https://github.com/grafana/grafana/blob/5f6616ff3eb2fb18c84af388b152394a67068309/pkg/services/preference/model.go#L14) is in the singular.